examples/Analytics.UpdateAction-sample.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Analytics.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Analytics.v1" Alias="Analytics" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.xml">
+    <edmx:Include Namespace="Org.OData.Aggregation.V1" Alias="Aggregation" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/examples/Org.OData.Aggregation.V1.SalesModel-sample.xml">
+    <edmx:Include Namespace="org.example.odata.salesservice" Alias="SalesModel" />
+  </edmx:Reference>
+
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="example.Analytics.UpdateAction" Alias="this">
+      <Action Name="UpdateAggregateSales">
+        <Annotation Term="Core.Description" String="Updates an aggregate entity of type `SalesModel.Sale`" />
+        <Parameter Name="Aggregate" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="String identifying the aggregate entity to be updated" />
+        </Parameter>
+        <Parameter Name="Updates" Type="SalesModel.Sale" Nullable="false">
+          <Annotation Term="Core.Description" String="Aggregate entity with nested properties that are to be updated, like the body of a PATCH request" />
+        </Parameter>
+        <ReturnType Type="SalesModel.Sale" Nullable="false">
+          <Annotation Term="Core.Description" String="Aggregate entity with updates applied" />
+        </ReturnType>
+      </Action>
+      <EntityContainer Name="container">
+        <EntitySet Name="Sales" EntityType="SalesModel.Sale">
+          <Annotation Term="Aggregation.CustomAggregate" Qualifier="Amount" String="Edm.Decimal" />
+        </EntitySet>
+        <ActionImport Name="UpdateAggregateSales" Action="this.UpdateAggregateSales" EntitySet="Sales" />
+      </EntityContainer>
```

Comment:
This won't work, the action import must be in the same service / entity container as the entity set being aggregated.

Reply:
Both entity set and action import are now in the entity container `this.container`. Only the entity type is in a different service.

Reply:
I'd prefer having the entity type also in this example, especially since the URL https://oasis-tcs.github.io/odata-vocabularies/examples/Org.OData.Aggregation.V1.SalesModel-sample.xml results in 404 Not Found.